Css/WebScrap.Css/Listeners/Helpers/AttributesComparer.cs

```diff
@@ -1,53 +1,37 @@
 namespace WebScrap.Css.Listeners.Helpers;
 
 internal class AttributesComparer(
```

Comment:
This class looks foggy.
Maybe some tests will help.

Reply:
Maybe, a wrapper around lookups. It's an Attribute list, man! Lets extract an entity and add a built-in behaviour.